node/pkg/alephium/watcher.go

```diff
@@ -236,14 +232,13 @@ func (w *Watcher) fetchEvents(ctx context.Context, logger *zap.Logger, client *C
 						errC <- err
 						return
 					}
+					logger.Info("received a message", zap.String("txId", unconfirmed.TxId), zap.String("blockHash", unconfirmed.BlockHash))
 					if unconfirmed.msg.IsAttestTokenVAA() {
-						logger.Info("received an attest token message", zap.String("txId", unconfirmed.TxId), zap.String("blockHash", unconfirmed.BlockHash))
```

Comment:
Q: Does it make sense to add a `zap.String("type", "attest")` or `zap.String("type", "transfer")` in log to distinguish the log entry as before? maybe it helps with debugging?